sdk/modules/ui/cpp/src/base_window.cpp

```diff
@@ -157,7 +165,11 @@ BaseWindow::~BaseWindow() {
     if (glfw_window_) {
         glfwMakeContextCurrent(glfw_window_);
         glDeleteBuffers(1, &vertex_buffer_);
-        glDeleteVertexArrays(1, &vertex_array_id_);
+ #if defined(__APPLE__) && !defined(__linux__)
+    glDeleteVertexArraysAPPLE(1, &vertex_array_id_);
```

Comment:
Same remark as with the other functions.

Reply:
Addressed in the previous comments.